src/main/resources/std_llvm_include.ll

```diff
@@ -167,5 +167,62 @@ define void @array_bounds_check({ i64, [0 x i8*] }* %array, i64 %index) {
     bounds.success:
         ret void
 }
+
+; Compare two strings for equality
+; Function Attrs: nounwind uwtable
+define zeroext i1 @scmp(i8* %a, i8* %b) #0 {
```

Comment:
Not really. But I think it would be nice to provide the C source (as a comment?) in case some bugfixes or modifications are necessary.